### PR TITLE
[K4.0] Crypsis : When ReCaptcha plugin is enabled registered users can't write #3173

### PR DESCRIPTION
**Pull request by xillibit**

### CODE REVIEW

File: components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini [hunk 1]
```diff
@@ -511,6 +509,11 @@ COM_KUNENA_CONFIGURATION_TWITTER_API_CONSUMER_SECRET_LABEL="The Twitter consumer
 COM_KUNENA_CONFIGURATION_TWITTER_API_CONSUMER_SECRET_DESC="The Twitter consumer secret to connect to API"
 COM_KUNENA_ADMIN_CONFIG_MAX_LINKS="Max Links in message"
 COM_KUNENA_ADMIN_CONFIG_MAX_LINKS_DESC="Enter here the max links in a message to prevent spam"
+COM_KUNENA_CONFIGURATION_CAPTCHA_FOR_WHO_TITLE="For who the captcha should be displayed ?"
```
Comment (810) [inline components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini]: should be completely

File: components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini [hunk 2]
```diff
@@ -511,6 +509,11 @@ COM_KUNENA_CONFIGURATION_TWITTER_API_CONSUMER_SECRET_LABEL="The Twitter consumer
...
+COM_KUNENA_CONFIGURATION_CAPTCHA_FOR_WHO_DESC="Choose if guest or registered users can be see captcha or nobody to disable it completly"
```
Comment (810) [inline components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini]: should be completely

File: components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini (hunk 1 above)
Comment (810) [inline components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini]: Should be:
Display Captcha for

**Comment by 810** [comment]
CAPTCHA Challenge for Users doesn't work anymore,

When you want captcha for guests and users until "Option"

So I set:
For who the captcha should be displayed  = guest
CAPTCHA Challenge for Users = 10
user post count = 5

**Comment by rich20** [comment]
Now you have Captcha either for guests or registered users. Captcha for both does not work.

**Comment by xillibit** [comment]
> Now you have Captcha either for guests or registered users. Captcha for both does not work.

I haven't noticed that it can work for both in the same time

**Comment by xillibit** [comment]
On blue eagle : 

- [x] For guests 
- [x] For registred users which has less posts than setting CAPTCHA Challenge for Users

On crypsis : 

- [x] For guests 
- [x] For registred users which has less posts than setting CAPTCHA Challenge for Users

File: components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini [hunk 3]
```diff
@@ -511,6 +509,11 @@ COM_KUNENA_CONFIGURATION_TWITTER_API_CONSUMER_SECRET_LABEL="The Twitter consumer
 COM_KUNENA_CONFIGURATION_TWITTER_API_CONSUMER_SECRET_DESC="The Twitter consumer secret to connect to API"
 COM_KUNENA_ADMIN_CONFIG_MAX_LINKS="Max Links in message"
 COM_KUNENA_ADMIN_CONFIG_MAX_LINKS_DESC="Enter here the max links in a message to prevent spam"
+COM_KUNENA_CONFIGURATION_CAPTCHA_FOR_WHO_TITLE="Display Captcha for guest, registered or both ?"
```
Comment (810) [inline components/com_kunena/admin/language/en-GB/en-GB.com_kunena.views.ini]: Change it just to:
Display Captcha for 

because you can't select both

**Comment by rich20** [comment]
These two together had previously always works.

CAPTCHA Challenge for Guests = Yes
and
CAPTCHA Challenge for Users = 80

**Comment by 810** [comment]
Not working when:
Display Captcha for : guest
CAPTCHA Challenge for Users: 40
User posts count: 20

Login as user, reply/new topic. I can't see the captcha

**Comment by rich20** [comment]
In Blue Eagle it works for both together.

**Comment by xillibit** [comment]
it works for me as a guest:

guest_e

or as an user:

user_e

**Comment by rich20** [comment]
It works also for me as guest or as user.  But I must change the setting "Display Captcha for : Guest Users" or "Display Captcha for : Registered Users".
When selected is "Display Captcha for : Guest Users" works the captcha not for registered users. 
It has with previous versions always functioned, Captcha for gusts and registered users at the same time. In Blue Eagle it works.

**Comment by xillibit** [comment]
Try now

**Comment by 810** [comment]
yes now it worked.

**Comment by rich20** [comment]
- [x] Success

**Pull request merged**